user/libeconf/patches/meson.patch

```diff
@@ -0,0 +1,14 @@
+--- a/meson.build	2024-09-18 07:53:14.000000000 -0300
++++ b/meson.build	2024-09-20 06:48:05.316117427 -0300
+@@ -77,11 +77,6 @@
+   version : meson.project_version(),
+ )
+ 
+-libeconf_static = static_library('econf',
```

Comment:
you can delete this patch and do -Ddefault_library=shared in meson args, pretty sure